velox/vector/tests/VectorTest.cpp

```diff
@@ -1275,9 +1269,18 @@ TEST_F(VectorTest, compareNan) {
 class VectorCreateConstantTest : public VectorTest {
  public:
   template <TypeKind KIND>
-  void testPrimitiveConstant(typename TypeTraits<KIND>::NativeType val) {
+  void testPrimitiveConstant(
+      typename TypeTraits<KIND>::NativeType val,
+      const TypePtr& type = {}) {
```

Comment:
perhaps, change the default to `Type::create<KIND>()` and simplify expectedStr logic below:

```
auto expectedStr = fmt::format(
        "[CONSTANT {}: {} value, {} size]",
        type->toString(),
        baseVector->toString(0),
        size_);
```